sfge/src/main/java/com/salesforce/rules/ops/cache/CacheUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.salesforce.rules.ops.cache;
+
+public final class CacheUtil {
+    private CacheUtil() {}
+
+    /*public void getFilesInPath(ApexPath path) {
```

Comment:
Is this commented out on purpose?

Reply:
Good catch. This file shouldn't be here.